extensions/commands/art/cmd_build_info.py

```diff
@@ -61,17 +61,17 @@ def _get_hashes(file_path):
 
 
 def _get_formatted_time():
-    now = datetime.datetime.now(datetime.timezone.utc)
-    local_tz_offset = now.astimezone().strftime('%z')
-    formatted_time = now.strftime('%Y-%m-%dT%H:%M:%S.%f')[:-3] + local_tz_offset
+    now = datetime.datetime.now().astimezone()
+    ms = now.microsecond // 1000  # convert to milliseconds
+    formatted_time = now.strftime(f"%Y-%m-%dT%H:%M:%S.{ms:03d}%z")
 
     # Apparently if the timestamp has the Z the BuildInfo is not correctly identified in Artifactory
     # if local_tz_offset == "+0000":
     #    formatted_time = formatted_time[:-5] + "Z"
 
     # from here: https://github.com/jfrog/build-info-go/blob/9b6f2ec13eedc41ad0f66882e630c2882f90cc76/buildinfo-schema.json#L63
-    if not re.match(r'^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{3}(Z|[+-]\d{4})$', formatted_time):
-        raise ValueError("Time format does not match BuildInfo required format.")
+    assert re.match(r'^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{3}(Z|[+-]\d{4})$', formatted_time), \
```

Comment:
Looks good, bit note that upstream has this same bug! https://github.com/jfrog/build-info-go/blob/main/buildinfo-schema.json#L63 shows
`^\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}.\\d{3}(Z|[+-]\\d{4})$`